CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Table of Contents
 
 ### Improvements
 
+* The default of how screenshots are created has changed from full page to viewport only. These two options are now available to set globally via properties and per test individually via `RecheckWebOptions`.
```

Comment:
I would mention this for breaking change, as this may potentially cause differences for users.

Secondly, this is really a new feature. While this was technically possible before, it was never exposed to the user directly.